04 Implementation/src/entities/Aendringstype.java

```diff
@@ -0,0 +1,7 @@
+package entities;
+
+public enum Aendringstype {
+    PROCENTAENDRING,
+    BELOEBMAESSIGT_AENDRING
```

Comment:
```suggestion
    INGEN,
    PROCENTAENDRING,
    BELOEBMAESSIG_AENDRING
```
Lad os gøre det muligt at der ingen ændring er.
+ Fix typo.

Reply:
Du glemte min ændring...